File: in1000/host2020/uke2/differanse.py
# 02.03: Differansen mellom to heltall

print("| Skriv inn to heltall | \n")

# int() sikrer at verdien som oppgis regnes som et heltall
x = int(input("Oppgi verdien til x: "))
y = int(input("Oppgi verdien til y: "))

print("Differansen mellom x og y er", x-y)

# For å finne absoluttverdien kan man legge inn "x-y" som argument i funksjonen
# "abs()"

print("Differansen/absoluttverdien mellom x og y er", abs(x-y))

File: inf1000/uke1/Differanse.java
/**
<h1> 1.03: Differansen mellom to heltall </h1>
*/

import java.util.Scanner;

class Differanse {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.println("Oppgi verdien til x: ");
        int x = Integer.parseInt(in.next());

        System.out.println("Oppgi verdien til y: ");
        int y = Integer.parseInt(in.next());

        int sum = x+y;
        System.out.println("Differansen mellom x og y er " + sum);
    }
}
